public/app/features/plugins/admin/components/PluginScreenshotCarousel.tsx

```diff
@@ -0,0 +1,217 @@
+import { css, cx } from '@emotion/css';
```

Comment:
There's nothing really that ties this carousel to plugins,. it is simply an images carousel so we can make this a generic component for images carousel instead.